parsl/executors/taskvine/executor.py

```diff
@@ -242,53 +242,53 @@ def start(self):
         logger.debug("Starting TaskVineExecutor")
 
         # Create a process to run the TaskVine manager.
-        submit_process_kwargs = {"ready_task_queue": self.ready_task_queue,
-                                 "finished_task_queue": self.finished_task_queue,
-                                 "should_stop": self.should_stop,
+        submit_process_kwargs = {"ready_task_queue": self._ready_task_queue,
+                                 "finished_task_queue": self._finished_task_queue,
+                                 "should_stop": self._should_stop,
                                  "manager_config": self.manager_config}
-        self.submit_process = multiprocessing.Process(target=_taskvine_submit_wait,
-                                                      name="TaskVine-Submit-Process",
-                                                      kwargs=submit_process_kwargs)
+        self._submit_process = multiprocessing.Process(target=_taskvine_submit_wait,
+                                                       name="TaskVine-Submit-Process",
+                                                       kwargs=submit_process_kwargs)
 
         # Create a process to run the TaskVine factory if enabled.
-        if self.use_factory:
-            factory_process_kwargs = {"should_stop": self.should_stop,
+        if self.worker_launch_method == 'factory':
+            factory_process_kwargs = {"should_stop": self._should_stop,
                                       "factory_config": self.factory_config}
-            self.factory_process = multiprocessing.Process(target=_taskvine_factory,
-                                                           name="TaskVine-Factory-Process",
-                                                           kwargs=factory_process_kwargs)
+            self._factory_process = multiprocessing.Process(target=_taskvine_factory,
+                                                            name="TaskVine-Factory-Process",
+                                                            kwargs=factory_process_kwargs)
 
         # Run thread to collect results and set tasks' futures.
-        self.collector_thread = threading.Thread(target=self._collect_taskvine_results,
-                                                 name="TaskVine-Collector-Thread")
+        self._collector_thread = threading.Thread(target=self._collect_taskvine_results,
+                                                  name="TaskVine-Collector-Thread")
         # Interpreter can exit without waiting for this thread.
-        self.collector_thread.daemon = True
+        self._collector_thread.daemon = True
 
         # Begin work
-        self.submit_process.start()
+        self._submit_process.start()
 
         # Run worker scaler either with Parsl provider or TaskVine factory
-        if self.use_factory:
-            self.factory_process.start()
+        if self.worker_launch_method == 'factory':
+            self._factory_process.start()
```

Comment:
is this `if` statement still ok when worker_launch_method is something other than `factory` or `provider`? maybe this doesn't need to call `initialize_scaling` with `manual` mode?